api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -11,19 +11,30 @@ import "gogoproto/gogo.proto";
 // [#protodoc-title: RBAC]
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
+// RBAC filter config.
+// User sees the shadow rules as replacement to the enforced rules.
+// For example, policy A runs in prod in enforced mode, user wants
+// to test policy B; in the config, enforced_rules contains {A.mode=enforced},
```

Comment:
`encorced_rules` -> `rules` in the comment?

Reply:
thanks, done.